relayer/chains/icon/events.go

```diff
@@ -8,8 +8,9 @@ import (
 
 // All the events
 const (
-	EmitMessage = "Message(str,int,bytes)"
-	CallMessage = "CallMessage(str,str,int,int,bytes)"
+	EmitMessage     = "Message(str,int,bytes)"
+	CallMessage     = "CallMessage(str,str,int,int,bytes)"
+	ExecuteRollback = "ExecuteRollback(int)"
```

Comment:
Same here should be rollbackmessage to keep in sync with current implementation.